package.json

```diff
@@ -1028,8 +1029,8 @@
     "jsonwebtoken": "^9.0.2",
     "jsts": "^1.6.2",
     "kea": "^2.6.0",
-    "langchain": "^0.0.186",
-    "langsmith": "^0.0.48",
+    "langchain": "^0.1.30",
+    "langsmith": "~0.0.48",
```

Comment:
Previously with the `^` we were getting minor upgrades. The latest is [`0.1.14`](https://www.npmjs.com/package/langsmith) now, so this switch to `~` would be a downgrade, no? Can we move to `0.1.14`, or does this break existing tracing functionality?

We can upgrade `langsmith` and corresponding tracing in a separate PR if there's overhead here, but just curious about this `^`/`~`change.

Reply:
cannot recall, i will change it around and see if there is a difference